NEWS.md

```diff
@@ -1,5 +1,6 @@
 ## 2023-XX-XX v3.1.0-SNAPSHOT
 * [KAFKAWRAP-50](https://issues.folio.org/browse/KAFKAWRAP-50) KafkaConsumerWrapper.fetch Should Set Consumer to Resumed State
+* [MODDATAIMP-886](https://issues.folio.org/browse/MODDATAIMP-886) Create Kafka topics instead of relying on auto create
```

Comment:
we don't need this in news, changes refer to data import, no new functionality added to this library